content/docs/user-guide/testing/lytesttools/example-test-walkthrough.md

```diff
@@ -0,0 +1,16 @@
+---
+description: ' Walk through and explanation of the LyTestTools example test. '
+title: LyTestTools Example Test Walkthrough
+linktitle: LyTestTools Example Test
+---
+
+Instructions on how to get started with an example test using the LyTestTools (LTT) testing package.
```

Comment:
This isn't a complete sentence. Maybe, "These are instructions..."?

Reply:
fixed